app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError, NotWearingMaskError)
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except NotVaccinatedError:
+            return "All friends should be vaccinated"
```

Comment:
Consider making the error message more specific for `NotVaccinatedError`, such as 'A friend is not vaccinated.'